category/category.go

```diff
@@ -0,0 +1,31 @@
+package category
+
```

Comment:
Are you sure that category identifier should be hard coded in database and in code?

Reply:
I'd prefer to seed table from the code, but do not want to spend time now. 
I've created separate issue for it https://github.com/lungria/spendshelf-backend/issues/43